README.md

```diff
@@ -1,2 +1,67 @@
-# RFCs
-RFCs for the Open 3D Foundation products
+[O3DE RFCs]: #O3DE-rfcs
+
+Many changes, such as bug fixes and documentation improvements can be
+implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes are "substantial", and we ask that these be put through a
+bit of a design process and produce a consensus among the O3DE community and
+the SIGs.
+
+The "RFC" (request for comments) process is intended to provide a consistent
+and controlled path for new features to be added to O3DE.
+
+## Table of Contents
+[Table of Contents]:
```

Comment:
This is missing the anchor link title (#table-of-contents). So the anchor link is just displaying text on the page.